README.md

```diff
@@ -0,0 +1,32 @@
+# termcord
```

Comment:
It's a great idea to make the first thing in your README the import path to use your library! (That is, after all, probably the first thing that people will want to know.)